docs/tutorials/build-a-dapp/pallet.md

```diff
@@ -89,10 +95,15 @@ Things like events, storage, and callable functions may look familiar to you if
 blockchain development. We will show you what each of these components looks like for a basic Proof
 Of Existence pallet.
 
-### Imports and Dependencies
+In the following sections, we have the required code blocks you can **copy and paste** into your
```

Comment:
Reads a little convoluted to me.
**Rephrase suggestion:** In the following sections, the code blocks required for the PoE pallet are provided for you to simply **copy and paste** into the empty `pallet/template/src/lib.rs` file.

Reply:
go ahead and change as you see fit!